driver/src/test/scala/spec/GraphSpec.scala

```diff
@@ -18,6 +19,15 @@ class GraphSpec extends AsyncWordSpec with Matchers {
         s.token shouldNot be("")
       }
     }
+//    "create the database" ignore  {
```

Comment:
if you're going to add support for creating a database, you should also add support for dropping one or this is an incomplete PR